BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpScreenAssembly.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SignUpScreenAssembly.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 27/07/2023.
+//
+
+import UIKit
+
+class SignUpScreenAssembly: StoryAssembly {
+    private let nextScreen: StoryAssembly
```

Comment:
This is signInAssembly

Reply:
Renamed to SignInDataAssembly (as class was named in pr #10)